src/main/java/com/andersen/entity/User.java

```diff
@@ -1,54 +1,38 @@
 package com.andersen.entity;
 
+import jakarta.persistence.*;
+import lombok.*;
+
 import java.time.LocalDateTime;
 import java.util.ArrayList;
 import java.util.List;
 
+import static jakarta.persistence.CascadeType.ALL;
+import static jakarta.persistence.FetchType.LAZY;
+
+@MappedSuperclass
+@Setter
+@Getter
+@ToString(exclude = "tickets")
+@NoArgsConstructor
+@RequiredArgsConstructor
 public abstract class User extends AbstractEntity {
 
+	@NonNull
+	@Column(name = "name")
 	protected String name;
 
+	@NonNull
+	@Column(name = "creation_date")
 	protected LocalDateTime creationDate;
 
+	@OneToMany(mappedBy = "client", fetch = LAZY, cascade = ALL, orphanRemoval = true)
```

Comment:
Do you have here LazyInitializationException? Or its works fine?

Reply:
> Do you have here LazyInitializationException? Or its works fine?

I don't have LazyInitializationException when the tickets field is excluded from @ToString lombok annotation, this is why I had to exclude it, otherwise I have

Reply:
ok) Also You can use fetch = EAGER